tests/storage/test_write.py

```diff
@@ -0,0 +1,79 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import time
+from pathlib import Path
+from IOUtils import ReadWriteOp
+import sys
+
+sys.path.append("..")
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from TestBase import Test
+import argparse
+
+extension = ".txt"
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--file_size", dest="file_size",
+                        required=True,
+                        help='Size of the files, eg - 1M FOR 1MB, 2K for 2KB')
+    parser.add_argument("--dest", dest="path",
+                        required=True,
+                        help='Specify the path')
+    args = parser.parse_args()
+    return args
+
+
+class Write(Test):
+    """Writes to a file (of a given size) in the specified path"""
+
+    def __init__(self, input_size, dest_path):
+        self.input_size = input_size
+        self.storage_path = dest_path
+        self.file_path = os.path.join("/", dest_path)
+        self.ops = ReadWriteOp()
+        self.ref_test_name = "write"
+        self.params = {}
+        self.params['file_size'] = self.input_size
+        self.params['output_folder'] = self.file_path
+        super().__init__(**self.params)
+
+    def write_test(self, input_size):
+        self.log.write("info", "Creating workload...")
+        payload = self.ops.generate_payload(1, input_size)
+        self.log.write("info", "Workload created")
+        self.log.write("info", "Begin of write operations...")
+        for files, content in enumerate(payload):
+            name = files
+            file_name = str(name) + extension
```

Comment:
"extension" variable is being used here.  I was planning to take only path (where the file would be written and the path doesn't include the file name) and file size from the user.

Reply:
Sure, makes sense.
You can even consider writing a file without extension